app/components/pipeline/modal/search-event/component.js

```diff
@@ -8,53 +8,73 @@ export default class PipelineModalSearchEventComponent extends Component {
 
   @service shuttle;
 
-  @tracked sha;
+  @tracked searchField = null;
```

Comment:
```suggestion
  @tracked searchField = 'message';
```
Let's set `message` as the default and maybe event set a constant for it to keep the default value in sync for the entire class.